flink-runtime-web/web-dashboard/src/app/pages/job-manager/log-detail/job-manager-log-detail.component.ts

```diff
@@ -35,20 +40,24 @@ import { EditorOptions } from 'ng-zorro-antd/code-editor/typings';
   styleUrls: ['./job-manager-log-detail.component.less']
 })
 export class JobManagerLogDetailComponent implements OnInit, OnDestroy {
+  public editorOptions: EditorOptions;
   public logs = '';
   public logName = '';
   public downloadUrl = '';
   public isLoading = false;
   public isFullScreen = false;
-  public editorOptions: EditorOptions = flinkEditorOptions;
 
   private readonly destroy$ = new Subject<void>();
 
   constructor(
     private readonly jobManagerService: JobManagerService,
     private readonly cdr: ChangeDetectorRef,
-    private readonly activatedRoute: ActivatedRoute
-  ) {}
+    private readonly activatedRoute: ActivatedRoute,
+    private readonly configService: ConfigService,
+    @Inject(JOB_MANAGER_MODULE_CONFIG) readonly moduleConfig: JobManagerModuleConfig
+  ) {
+    this.editorOptions = moduleConfig.editorOptions || JOB_MANAGER_MODULE_DEFAULT_CONFIG.editorOptions;
```

Comment:
Why do we need this fallback to the non-injected version?

Reply:
This is because `editorOptions` can be omitted if someone overrides the whole `moduleConfig` through the token.